src/views/home/components/header/header.styles.ts

```diff
@@ -4,52 +4,43 @@ import { Theme } from "src/styles/theme";
 
 const useHeaderStyles = createUseStyles((theme: Theme) => ({
   header: {
-    display: "flex",
-    fontWeight: theme.fontWeights.medium,
-  },
-  settingsButton: {
-    flex: 1,
     display: "flex",
     alignItems: "center",
+    marginTop: theme.spacing(2),
   },
-  myAccountButton: {
+  logoWrapper: {
     flex: 1,
-    display: "flex",
     alignItems: "center",
-    justifyContent: "right",
+    textAlign: "center",
+    flexDirection: "column",
+  },
+  appName: {
+    marginTop: theme.spacing(0.5),
+    color: theme.palette.black,
   },
-  button: {
-    display: "inline-flex",
+  link: {
+    display: "flex",
     alignItems: "center",
-    padding: [[theme.spacing(0.75), theme.spacing(1)]],
+    padding: [theme.spacing(0.75), theme.spacing(1)],
     borderRadius: theme.spacing(1),
-    height: theme.spacing(4.5),
+    transition: theme.hoverTransition,
     "&:hover": {
-      backgroundColor: theme.palette.grey.light,
+      backgroundColor: theme.palette.grey.main,
     },
   },
-  icon: {
-    width: 24,
-  },
-  text: {
-    margin: [[0, theme.spacing(1)]],
+  settingsLabel: {
     display: "none",
     [theme.breakpoints.upSm]: {
-      display: "inherit",
+      display: "block",
+      marginLeft: theme.spacing(1),
     },
   },
-  logoBox: {
-    flex: 1,
-    alignItems: "center",
-    textAlign: "center",
-    flexDirection: "column",
-  },
-  logoIcon: {
-    width: 43,
-  },
-  logoText: {
-    marginTop: -theme.spacing(1.5),
-    fontSize: theme.spacing(1.75),
+  activityLabel: {
```

Comment:
Why you don't use the same class to cover settingsLabel and activityLabel?

Reply:
They have different margins